data/meson.build

```diff
@@ -7,6 +7,11 @@ install_data('its/metainfo.its',
 install_data('its/metainfo.loc',
               install_dir: join_paths(get_option('datadir'), 'gettext', 'its'))
 
+# Do not rely on an exe wrapper for rel-info, use the system one in that case
+if meson.is_cross_build()
+    ascli_exe = find_program('appstreamcli')
```

Comment:
This would fetch `appstreamcli` from the environment - is that really needed? Reason is that, since ascli comes from AppStream, it may not exist in the environment, or may even exist as an older version. We often use newer features in AppStream itself first, so it is quite important that AppStream validates its own files with the tools it has just built, and doesn't fail due to using older tools from the environment.

Reply:
It's only needed for cross-compilation. Otherwise we would need to build a native version too - which is a lot more complicated, and should proabably be done as a sub-build somehow... tricky

Reply:
> Iow, it's a lot simpler to install a native version before doing a cross-compilation. If you worry about versions, we could check the native appstream.pc version..

That would be nice to or, or maybe at least emit a warning in the log file, so debugging any issues becomes a bit easier :-)

Reply:
added